pyslim/spatial.py

```diff
@@ -3,61 +3,103 @@
 import warnings
 import numpy as np
 
-# probably don't need all of these
 from .slim_tree_sequence import *
-from .slim_metadata import *
-from .provenance import *
-from .util import *
 
 
-def in_location_bin(locations, x0, x1, y0, y1):
+def _in_location_bin(locations, x0, x1, y0, y1):
```

Comment:
underscores at the start of the functions make it so this method won't be exported by pyslim - and, I don't think we want to be making this a public function; we just want to use it in this file. Same with average_time_alive.